components/bt/host/bluedroid/btc/profile/std/sdp/btc_sdp.c

```diff
@@ -331,7 +327,13 @@ static int add_raw_sdp(const bluetooth_sdp_record* rec)
         sdp_handle = 0;
         BTC_TRACE_ERROR("%s() FAILED, status = %d", __func__, status);
     } else {
-        bta_sys_add_uuid(service);
+        if (rec->hdr.bt_uuid.len == ESP_UUID_LEN_16){
+            bta_sys_add_uuid(rec->hdr.bt_uuid.uuid.uuid16);
```

Comment:
The uuid should also be removed when [deleting](https://github.com/espressif/esp-idf/blob/28167ea5a3cdd49227216c4131a9df7f4eda260e/components/bt/host/bluedroid/btc/profile/std/sdp/btc_sdp.c#L828) the sdp record.